mobile/app/Components/UserSignUpForm.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { View, Button, TextInput } from 'react-native';
+// import axios from 'axios';
+// USECLIENT TO CHANGE STATE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+interface whatever {
+  onChangeUser: (email, demographics, name, phoneNum, community) => void
+}
+
+export default function UserSignUpForm ({ onChangeUser }: whatever) {
+
+  const [email, onChangeEmail] = React.useState('');
+  const [demographics, onChangeDemo] = React.useState('');
+  const [name, onChangeName] = React.useState('');
+  const [phoneNum, onChangePhoneNum] = React.useState('');
+  const [community, onChangeCommunity] = React.useState('');
+
+const handleAddUser = () => {
+  onChangeUser(email, demographics, name, phoneNum, community);
```

Comment:
You can replace this line with an HTTP request to add the user directly by making an HTTP request to the user endpoint. Thus, you don't need the `onChangeUser` property to be passed. 

In the event where you do need props, you must also validate the props for the provided component (as you may have done in your developer onboarding).

Reply:
resolved